clang/docs/ReleaseNotes.rst

```diff
@@ -494,6 +494,7 @@ Bug Fixes to C++ Support
 - Fix a crash when parsing a pseudo destructor involving an invalid type. (#GH111460)
 - Fixed an assertion failure when invoking recovery call expressions with explicit attributes
   and undeclared templates. (#GH107047, #GH49093)
+- During the lookup for a base class name, non-type names are ignored.
```

Comment:
```suggestion
- During the lookup for a base class name, non-type names are ignored. (#GH16855)
```
Almost forgot about this: we typically include the number of the issue that a change fixes, if there is one.